static/src/javascripts/es6/projects/common/modules/crosswords/focussedClue.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export default React.createClass({
+    render: function () {
+        if(this.props.focussedClue === null) {return false}
+
+        let classList = {
+            'crossword__focussed-clue--direction': true,
+            'crossword__focussed-clue--direction-down': this.props.focussedClue.direction === 'down',
+            'crossword__focussed-clue--direction-across': this.props.focussedClue.direction === 'across'
+        };
+        return <div className='crossword__focussed-clue-wrapper'>
+            <div className='crossword__focussed-clue'>
+                <div className={classNames(classList)}>{this.props.focussedClue.number} {this.props.focussedClue.direction}</div>
+                {this.props.focussedClue.clue}
+            </div>
+            {this.props.contextualClues.map((contextualClue) => {
+                let classList = {
+                    'crossword__contextual-clue--direction': true,
+                    'crossword__contextual-clue--direction-down': contextualClue.direction === 'down',
+                    'crossword__contextual-clue--direction-across': contextualClue.direction === 'across'
+                };
+                const focusClue = () =>
+                    this.props.focusClue(contextualClue.position.x,
+                                         contextualClue.position.y,
+                                         contextualClue.direction);
+                return <button className='crossword__contextual-clue u-button-reset'
+                               onClick={focusClue}>
+                    <div className={classNames(classList)}>{contextualClue.number} {contextualClue.direction}</div>
+                    {contextualClue.clue}
+                </button>;
+            })}
+        </div>;
+    }
+});
+
```

Comment:
@rich-nguyen and I did some research into the `jsx` loader plugin for jspm (https://github.com/floatdrop/plugin-jsx). What's really nice is how the [`translate` method](https://github.com/floatdrop/plugin-jsx/blob/6f0f35a62b6a6231ed5eaa5b675150291db25521/jsx.js#L5) is used for runtime compiling and bundling. During the bundling process, jspm will save the output of the compile in the bundle.